alterconfigs_test.go

```diff
@@ -0,0 +1,34 @@
+package kafka
+
+import (
+	"context"
+	"testing"
+)
+
+func TestClientAlterConfigs(t *testing.T) {
+
+	client, shutdown := newLocalClient()
+	defer shutdown()
+
+	topic := makeTopic()
+	createTopic(t, topic, 1)
+	defer deleteTopic(t, topic)
+
+	res, err := client.AlterConfigs(context.Background(), &AlterConfigsRequest{
+		Resources: []AlterConfigRequestResource{{
+			ResourceType: ResourceTypeTopic,
+			ResourceName: topic,
+			Configs: []AlterConfigRequestConfig{{
+				Name:  "max.message.bytes",
+				Value: "200000",
+			},
+			},
+		}},
+	})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Logf("Client.DescribeConfigs() = %v", res)
```

Comment:
could we make some assertions on the contents of `res`?

I prefer the testify libs
```
"github.com/stretchr/testify/assert"
"github.com/stretchr/testify/require"
```

maybe something like
```
describeResp, err := client.DescribeConfigs(...)
assert.Equal(t, describeResp.Resources, res.Resources)
```